### PR TITLE
Chore/quick cleanup

### PR DESCRIPTION
**Pull request by hhamud**
---
name: Pull Request
about: Create a pull request for this project

---

## Description
* Cleanup of some hooks and typescript generated files
* changes were building up so I wanted to send this off for review before further work

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by rossneilson — APPROVED** [review]
Nice work, good to do this occasionally for small issues we notice

File: src/hooks/useFixedPriceSaleCommitmentsByBuyerId.ts
Comment (adamazad) [inline src/hooks/useFixedPriceSaleCommitmentsByBuyerId.ts:21]: _utils_ and _helpers_ imply same exports. Refactor.

File: src/hooks/useFixedPriceSaleCommitmentsByBuyerId.ts
Comment (adamazad) [inline src/hooks/useFixedPriceSaleCommitmentsByBuyerId.ts:38]:
```suggestion
export function useFixedPriceSaleCommitmentsByBuyerIdQuery(buyerId: string | undefined | null): UseSalesQueryResult {
```

File: src/redux/bids/index.ts
Comment (adamazad) [inline src/redux/bids/index.ts:81]:
```suggestion
export const updateCommitmentsSuccess = (payload: GetAllBidsBySales_fixedPriceSale_commitments) => ({
```

File: src/redux/bids/index.ts
Comment (adamazad) [inline src/redux/bids/index.ts:108]: Could you elaborate on this, please?

File: src/views/Sale/FairSale.tsx
Comment (adamazad) [inline src/views/Sale/FairSale.tsx:51]: Please refactor 

```typescript
export type FairBidPick = Pick<FairSaleBid, 'address' | 'tokenOut' | 'tokenIn'>

export type FixedPricePick = Pick<FixedPriceSalePurchase, 'amount' | 'buyer'>
```

`Pick` is vague, and confuses others

**Review by adamazad — CHANGES_REQUESTED** [review]
Good job using Apollo.

File: src/redux/bids/index.ts
Comment (hhamud) [reply src/redux/bids/index.ts:23]: I've kept it as bids since the redux component is supposed to work for all the upcoming sales. Plus the redux component will soon be deprecated anyway

File: src/redux/bids/index.ts
Comment (hhamud) [reply src/redux/bids/index.ts:81]: https://github.com/cryptonative-ch/mesa-interface/pull/393#discussion_r686366470

File: src/redux/bids/index.ts
Comment (hhamud) [reply src/redux/bids/index.ts:108]: checks for purchases with the same id between payload and state

Comment (hhamud) [reply src/redux/bids/index.ts:108]: a function that checks if a purchase event in the action payload is already in the redux state. This is to prevent duplicates from occurring as there are two sources of information in the interface; one that reads directly from the chain and one from the subgraph. 

I assume that there apollo has a better way of dealing with duplicates and caching.

**Comment by hhamud** [comment]
I will wait until tomorrow to merge this if there are any more comments.

File: src/hooks/useBids.ts
Comment (adamazad) [inline src/hooks/useBids.ts:40]: `allBids`. `totalBids` implies a count.

**Pull request merged**
